webapp/src/store/actions.ts

```diff
@@ -23,23 +23,24 @@ const actions = wrapActions({
       commit('addBeneficiary', bnf);
     }
   },
-  async createUser({ commit }, { phone, password }: { phone: string; password: string }) {
-    const user = await Users.createUser({ phone, password });
-    await Users.login({ phone, password });
+  async createUser({ commit }, { phone, password, googleIdToken }: { phone: string; password: string; googleIdToken: string }) {
+    const user = await Users.createUser({ phone, password, googleIdToken });
+    await Users.login({ phone, password, googleIdToken });
     commit('setUser', user);
     if (user) {
       router.push({ name: 'home' });
     }
   },
-  async signUserIn({ commit }, { phone, password }: { phone: string; password: string }) {
-    const user = await Users.login({ phone, password });
+  async signUserIn({ commit }, { phone, password, googleIdToken }: { phone: string; password: string; googleIdToken: string }) {
+    const user = await Users.login({ phone, password, googleIdToken });
     if (user) {
       commit('setUser', user);
       if (router.currentRoute.name !== 'home') router.push({ name: 'home' });
     }
   },
   async signUserOut({ dispatch }) {
     await Users.logout();
+    googleSignOut();
```

Comment:
Isn't this function supposed to be asynchronous? If so, you should place `await` in front of the call

Reply:
I had tried that and it said the 'await' was unnecessary. Let me look into it again...